cmd/fluxctl/install_cmd.go

```diff
@@ -23,22 +28,23 @@ func (opts *installOpts) Command() *cobra.Command {
 fluxctl install --git-url 'git@github.com:<your username>/flux-get-started' | kubectl -f -`,
 		RunE: opts.RunE,
 	}
-	cmd.Flags().StringVarP(&opts.GitURL, "git-url", "", "",
+	cmd.Flags().StringVar(&opts.GitURL, "git-url", "",
```

Comment:
All of these were using `*VarP`, the variant for providing a shortcut single-letter flag, then giving an empty string as the shortcut :man_shrugging:

Reply:
That was my bad, didn't really think about the alternatives